docs/apps/index.md

```diff
@@ -0,0 +1,69 @@
+---
+title: 🚀 Installation
+sidebar_position: 2
+tags:
```

Comment:
Please add more tags. It will help with indexing

Reply:
Examples:
- Apps
- Layout
- Description
- Tooltip
- Hover
- Line clamp
...